.eslintrc.cjs

```diff
@@ -4,17 +4,30 @@ module.exports = {
     root: true,
     extends: "eslint",
     env: {
-        es6: true
+        es2020: true
+    },
+    parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: "module"
     },
     overrides: [
         {
-            files: ["tests/lib/**"],
+            files: ["*.cjs"],
             parserOptions: {
-                ecmaVersion: 2020
-            },
+                sourceType: "script"
```

Comment:
@mysticatea doesn't [`plugin:node/recommended`](https://github.com/eslint/eslint/blob/8490fb42e559ef0b3c34ac60be4e05e0d879a9cb/packages/eslint-config-eslint/default.yml#L3) already handle this?